doc/testing.md

```diff
@@ -379,9 +379,12 @@ never more than *memory size in GB/2*.
 
 #### TIMEOUT_FACTOR
 
-The timeout factor (`-timeoutFactor`).
-
-Defaults to 4.
+The `TIMEOUT_FACTOR` is forwarded to JTReg framework itself
+(`-timeoutFactor`). Also, some test cases that programmatically wait a
+certain amount of time will apply this factor. If we run in
+interpreted mode (`-Xcomp`), [RunTest.gmk](../make/RunTests.gmk)
```

Comment:
Maybe
```suggestion
interpreted mode (`-Xint`), [RunTest.gmk](../make/RunTests.gmk)
```

Reply:
Thanks for catching this fault of mine. I will update the text and change `interpreted mode`, as it is really `-Xcomp` we are looking at in the RunTest.gmk.

Reply:
yep, let use Xcomp, the Xint is not really supported, a lot of tests might start failing